### PR TITLE
Fill out README with new sections and add deployment notes

### LINKED ISSUE #9: Add deployment notes to README
Add additional notes about how to deploy this on a live system.

- Environment variables

Switch to [this README template](https://github.com/wearehive/project-guidelines/blob/master/README.sample.md) as it has a much better structure to the current one.

### PR DESCRIPTION
**Pull request by dan1elhughes**
### Description

Rewrites the README for better information and structure, with deployment notes.

[View as actual document](https://github.com/dan1elhughes/ethereum-pos/blob/0a21f2e54ba4501d9e416a6fa9c041b252afd0a2/README.md).

### Issues fixed

Fixes #9.

### Checklist

- [x] `npm test` returns no warnings or errors.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,50 +1,65 @@
...
+A quick introduction of the minimal setup you need to get the app up & running.
+
+```shell
+git clone git@github.com:HeathWallace/ethereum-pos.git
+cd ethereum-pos
+npm install
+npm start
+```
+
+Create a file `.env` in the root of the app and add the variables shown in the console.
```
Comment (peasandwell) [inline README.md]: Could this step actually be automated...? Or is that too risque?

File: README.md [hunk 2]
```diff
@@ -1,50 +1,65 @@
...
+- [Node](https://nodejs.org)
 
-### Installing
+### Deploying / Publishing
 
-```
-- npm install
-- npm start
-```
+To deploy the app, build the app using the included build script (`npm run build`) and place the output of the `/build` directory in a publicly accessibly folder.
```
Comment (peasandwell) [inline README.md]: Typo on "accessible"

File: README.md (hunk 1 above)
Comment (ShaunBulbrook) [reply README.md]: The issue in automating this in the build promotes Gromits over anything else. (Where the application should work with anything) - We also want to keep our internal identity resolver separate, etc.

**Comment by dan1elhughes** [comment]
Rebased onto master

**Pull request merged**